recipes/grpc-proto/all/conanfile.py

```diff
@@ -59,11 +59,11 @@ def generate(self):
         tc.generate()
 
     def requirements(self):
-        self.requires('protobuf/3.21.4', transitive_headers=True)
+        self.requires('protobuf/3.21.9', transitive_headers=True)
```

Comment:
```suggestion
        self.requires('protobuf/3.21.9', transitive_headers=True, transitive_libs=True)
```

(I'm currently verifying if this is accurate, will post a reply with the results)